MagicalCryptoWallet/Helpers/ByteHelpers.cs

```diff
@@ -110,5 +110,20 @@ public static byte[] FromHex(string hex)
 
 			return bytes;
 		}
+
+		public static byte[] SafeSubarray(this byte[] array, int offset, int count)
+		{
+			if(array == null)
```

Comment:
Could you use the Guard class here? Get used to it: `Guard.NotNull(nameof(array), array);`

Reply:
Done